Assets/GlitchEscape/Scripts/Player/Mechanics/PlayerManifestController.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Controls;
+
+public class PlayerManifestController : MonoBehaviour, IPlayerControllerComponent
+{
+    const float GRAVITY = 9.81f; // m/s^2
+
+    private PlayerController controller;
+    private Player player;
+    private Input input;
+    private Animator animator;
+    private new Rigidbody rigidbody;
+    private List<Material> defaultMaterials;
+    private Renderer[] renderers;
+    //public Material glitchMaterial;
+
+    public void SetupControllerComponent(PlayerController controller)
+    {
+        this.controller = controller;
+        player = controller.player;
+        rigidbody = player.rigidbody;
+        animator = player.animator;
+        input = player.input;
+        input.Controls.Manifest.performed += context => {
+            bool pressed = context.ReadValue<float>() > 0f;
```

Comment:
this works; we should probably just add a full input abstraction or something though >_<

(don't worry about this; we could refactor this later)